setup.py

```diff
@@ -98,7 +98,7 @@
 
 setup(
     name="schema-salad",
-    version="8.0",  # update the VERSION prefix in the Makefile as well 🙂
+    version="8.1",  # update the VERSION prefix in the Makefile as well 🙂
```

Comment:
As the comment says, please `update the VERSION prefix in the Makefile as well`!